app/router.js

```diff
@@ -21,7 +29,22 @@ const resolveService = require('./middleware/resolve_service.js')
 exports.paths = paths
 
 exports.bind = function (app) {
-  'use strict'
+  AWSXRay.enableManualMode()
+  AWSXRay.setLogger(logger)
+  AWSXRay.middleware.setSamplingRules('aws-xray.rules')
+  AWSXRay.config([AWSXRay.plugins.ECSPlugin])
+  app.use(AWSXRay.express.openSegment('pay_frontend'))
```

Comment:
do we need a `app.use(AWSXRay.express.closeSegment())` after the definition of routes? See "Tracing Incoming Requests with Express" at https://docs.aws.amazon.com/xray/latest/devguide/xray-sdk-nodejs-middleware.html

Reply:
We do, good spot thanks :-)

Sorry, this PR was not ready for prime time. I've updated the title as such.

Reply:
sorry, i guess i'm eager to get this merged as it will help will perf testing ;) no worries though, i'm off tomorrow and am occupied with something else.